### PR TITLE
Better handle pp_check() S3 exporting

### PR DESCRIPTION
**Pull request by bwiernik**
Closes https://github.com/easystats/performance/issues/360

### CODE REVIEW

**Comment by strengejacke** [comment]
As I commented [here](https://github.com/easystats/performance/issues/360#issuecomment-913111540), I'd suggest removing the onLoad-stuff and just just our own name for that function. @DominiqueMakowski any thoughts from your side?

**Comment by bwiernik** [comment]
I don't think it's much of a problem to leave the S3 registration there

**Comment by strengejacke** [comment]
ok, so this doesn't require higher a R version (at least 3.6)?

**Comment by bwiernik** [comment]
No, it doesn't change our minimum version at all. 

I've found the clearerest and least disruptive approach I think. roxygen2 has a `@rawNamespace` command we can use to export the `if (getRversion() >= "3.6.0") { }` documentation. Then, we can simply say in the documentation that `On R 3.6.0 and higher, if \pkg{bayesplot} (or a package that imports \pkg{bayesplot} such as \pkg{rstanarm} or \pkg{brms}) is loaded, \code{pp_check()} is also available as an alias for \code{check_predictions()}`

**Comment by bwiernik** [comment]
@strengejacke I think we should merge this before CRAN release. Can I go ahead and do that?

**Comment by strengejacke** [comment]
Yes, we can merge this PR.

File: R/pp_check.R [hunk 1]
```diff
@@ -52,18 +49,20 @@
...
-#'   pp_check(model)
+#' model <- lm(mpg ~ disp, data = mtcars)
+#' if (require("see")) {
+#'   check_predictions(model)
 #' }
 #' @export
-pp_check <- function(object, ...) {
-  UseMethod("pp_check")
+check_predictions <- function(object, iterations = 50, check_range = FALSE, re_formula = NULL, ...) {
+  if (isTRUE(insight::model_info(object, verbose = FALSE))) {
```
Comment (strengejacke) [inline R/pp_check.R:58]: I just realized this code line here - what are you expecting from `model_info()` to be returned? I think this if-statement currently is never true.

Comment (strengejacke) [reply R/pp_check.R:58]: @bwiernik I think this should be checking for Bayesian methods, right?

**Pull request merged**